src/lib.rs

```diff
@@ -52,7 +52,7 @@
 //!     vec![(0f64, &1), (2f64, &0), (2f64, &2), (8f64, &3)]
 //! );
 //! ```
-
+#![feature(vec_remove_item)]
```

Comment:
I'm a bit rusty on Rust - could you share more context for this?

Reply:
This is so you can remove items from a vec by value instead of index. I actually left it in by mistake, I ended up use iter/filter instead. I'll remove it.